.pnp.loader.mjs

```diff
@@ -1,2047 +0,0 @@
-import { URL as URL$1, fileURLToPath, pathToFileURL } from 'url';
```

Comment:
Do we need to add this file to .gitignore (like .pnp.cjs), or does Yarn not generate it anymore?

Reply:
Yarn 4 does not generate this file anymore, only `.pnp.cjs` is generated. Maybe it has been included in Yarn 4 as well.

Reply:
Yarn still generates it if it detects that ESM support is required.